packages/cli/generators/controller/index.js

```diff
@@ -73,3 +148,90 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
     }
   }
 };
+
+/**
+ * Optional prompt to retrieve variables for artifacts.
+ * Not a member of the class because Yeoman will always call Generator
+ * prototype functions! Use .apply or .call with this function!
+ */
+function promptArtifactCrudVars() {
```

Comment:
We should make it a member function and check `this.artifactInfo.controllerType === ControllerGenerator.CRUD` to proceed.